distribution/src/conf/internal-apis.xml

```diff
@@ -33,6 +33,26 @@
                             </user>
                         </users>
                     </UserStore>
+                </handler-->
+                <handler class="org.wso2.micro.integrator.management.apis.security.handler.JWTTokenSecurityHandler"
+                         name="JWTTokenSecurityHandler">
+                    <TokenStoreConfig>
+                        <MaxSize>200</MaxSize>
+                        <TokenCleanupTaskInterval>600000</TokenCleanupTaskInterval>
+                        <RemoveOldestTokenOnOverflow>false</RemoveOldestTokenOnOverflow>
+                    </TokenStoreConfig>
+                    <TokenConfig>
+                        <expiry>3600000</expiry>
```

Comment:
Do we need to configure this in milliseconds? I don't think anybody will have a requirement to expire a token in some number of millis. Would changing it to seconds make more sence? WDYT?

Reply:
Thanks changed to seconds